cmd/container-disk-v2alpha/README.md

```diff
@@ -6,8 +6,7 @@ KubeVirt runtime.
 
 This Base Container is compatible with disk type ContainerDisk:v1alpha
 
-# ContainerDisk:v1alpha
-## Storing Disks in Container Registry
+# Storing Disks in Container Registry
```

Comment:
we're going to need some updates here in the user-guide.

https://kubevirt.io/user-guide/docs/latest/creating-virtual-machines/disks-and-volumes.html#containerdisk

Reply:
Done: https://github.com/kubevirt/user-guide/pull/273